Formula/frotz.rb

```diff
@@ -35,8 +34,8 @@ def install
   end
 
   test do
-    assert_match "FROTZ", shell_output("#{bin}/frotz --version").strip
-    assert_match "FROTZ", shell_output("#{bin}/sfrotz --version").strip
-    assert_match "FROTZ", shell_output("#{bin}/dfrotz --version").strip
+    assert_match "FROTZ", shell_output("#{bin}/frotz -v").strip
```

Comment:
Can these be replaced with tests that exercise some of the functionality of the app? 
Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
To do that would require a test file, but I can't figure out how to access one during the tests. 

I checked the https://docs.brew.sh/Formula-Cookbook but All of the test examples I see are configured to write an ASCII text file during the test, like the `cmake` test. https://github.com/Homebrew/homebrew-core/blob/master/Formula/cmake.rb

```rb
  test do
    (testpath/"CMakeLists.txt").write("find_package(Ruby)")
    system bin/"cmake", "."
  end
```

Specifically, there's a file checked into the frotz tarball that I'd like to access, `src/test/etude/etude.z5`. How would I access that in a Homebrew test?

Reply:
Never mind, I found one, and added a real test.